locales/en-US/formautofill.properties

```diff
@@ -31,3 +41,40 @@ fieldNameSeparator = ,\u0020
 # The text would be e.g. Also fill company, phone, email
```

Comment:
The example doesn't match the actual string. Also, in which context is phishingWarningMessage used instead of phishingWarningMessage2?